lib/console/deployments/observability/webhook/newrelic.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Console.Deployments.Observability.Webhook.Newrelic do
+  @behaviour Console.Deployments.Observability.Webhook
+  import Console.Deployments.Observability.Webhook.Base
+
+  def associations(:project, payload, acc) do
+    Map.put(acc, :project_id, extract_from_payload(payload, ~r/plrl_project:\s*([^,\s]+)/, &project/1))
+  end
+
+  def associations(:cluster, payload, acc) do
+    Map.put(acc, :cluster_id, extract_from_payload(payload, ~r/plrl_cluster:\s*([^,\s]+)/, &cluster/1))
+  end
+
+  def associations(:service, payload, %{cluster_id: id} = acc) when is_binary(id) do
+    Map.put(acc, :service_id, extract_from_payload(payload, ~r/plrl_service:\s*([^,\s]+)/, fn value ->
+      clean_value = String.replace(value, ~s("), "")
+      service(id, clean_value)
+    end))
+  end
+
+  def associations(_, _, acc), do: acc
+
+  def state(%{"state" => state}) do
```

Comment:
both state and severity should be implemented as mult-function head matches actually, eg:

```elixir
def state(%{"state" => "ACTIVATED"}), do: :firing
def state(_), do: :medium
```